### DISCUSSION TITLE
i am trying to login with my email id but not happening!!!!!

### DISCUSSION THREAD

**Discussion by Abhay-Agnihotri01** [vercel/community#3957] [Account Issue] [answered]
### Question

I am trying to login with my email id to the vercel platform and i tried all the login methods there but all the time it is saying that user is blocked.. how to fix this please assist on this isssue..

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble logging in. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by Abhay-Agnihotri01** [reply]
i didnt got any email so is there any way to contact the individual of the vercel platform

**Reply by amyegan** [reply]
Sorry for the delay. You can open a new support case via the [support form](https://vercel.com/support/request) and other [direct channels](https://vercel.com/guides/how-to-get-vercel-support#direct-channels).
